### ISSUE TITLE
Investigate integrating graphing functionality with ggplot

### ISSUE THREAD

**Issue by Hong-Revo** [cloudyr/AzureKusto#52]

**Comment by Hong-Revo** [comment]
@alexkyllo how hard do you think it would be to get Kusto's graphs into R? From what I can tell, if you submit a graphing command, the returned object contains the data as well as the svg image. We could use either display the svg, or try to mimic it using ggplot2.

**Comment by alexkyllo** [comment]
Simply saving the SVG into an object so that the user can access it, should be straightforward, but I'm not so sure about integrating it with ggplot2. You can save a ggplot as SVG, but that's a one-way conversion--you can't convert an SVG back into a ggplot object.

**Comment by Hong-Revo** [comment]
Yeah, I don't mean trying to pull apart the SVG -- rather, passing the data in the response to R's graphing functions. Probably too much trouble compared to just displaying the SVG though.
